backend/app/helpers/spree/admin/stores_helper.rb

```diff
@@ -12,6 +12,18 @@ def stores_dropdown_values
 
         formatted_stores
       end
+
+    def stores
```

Comment:
@Vegann this is already defined here: https://github.com/spree/spree/blob/master/backend/app/controllers/spree/admin/base_controller.rb#L57